src/custom/pages/Profile/index.tsx

```diff
@@ -75,7 +79,17 @@ export default function Profile() {
       <Wrapper>
         <GridWrap>
           <CardHead>
-            <Title>Affiliate Program</Title>
+            {account && refAddress.value && (
+              <FlexRow>
```

Comment:
It should be like this (without adding extra space upper the Affiliate Program title)
![image](https://user-images.githubusercontent.com/622217/154120005-07a54659-3556-47ff-b555-3495e9c76c39.png)

Reply:
In mobile, I agree with Elena, we should list "Referred by: ... " below the "Las updated..." data.